server.py

```diff
@@ -17,14 +18,25 @@
 async def create_url(request: Request):
     try:
         data = await request.json()
-        url = data["url"]
+        url = data.get("url", None)
         alias = data.get("alias", None)
-        if (alias is None):
-            alias = hash.create_alias(url)
+        if (url is None):  # url not provided
+            raise ValueError("Url not provided.")
+        if (helpers.alias_exists(DATABASE, alias)):  # alias exists
+            raise HTTPException(
```

Comment:
this can raise another ValueError saying "Alias not valid" to keep raising errors inside of the try block and raising the httpexception inside the except block